MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/DataProcessorUI/DataProcessorPresenter.h

```diff
@@ -1,10 +1,10 @@
 #ifndef MANTIDQTMANTIDWIDGETS_DATAPROCESSORPRESENTER_H
 #define MANTIDQTMANTIDWIDGETS_DATAPROCESSORPRESENTER_H
 
+#include <QVariant>
```

Comment:
Then I'm afraid that `std::map` needs the complete definition. I'd leave it as it is then.